.github/advanced-issue-labeler.yml

```diff
@@ -0,0 +1,49 @@
+# syntax - https://github.com/redhat-plumbers-in-action/advanced-issue-labeler#policy
+# <name of label>: [<list of keywords>]
+---
+
+policy:
+  analyze: ['systemd-analyze']
+  ask-password: ['systemd-ask-password']
+  binfmt: ['systemd-binfmt']
+  cgtop: ['systemd-cgtop']
+  cryptsetup: ['systemd-cryptsetup']
+  delta: ['systemd-delta']
+  env: ['systemd-env-generator', '30-systemd-environment-d-generator']
+  fsck: ['systemd-fsck']
+  gpt-auto: ['systemd-gpt-auto-generator']
+  growfs: ['systemd-growfs']
+  homed: ['systemd-homed', 'homectl', 'pam_systemd_home']
+  hostname: ['systemd-hostnamed', 'hostnamectl']
+  hwdb: ['the hardware database files']
+  import: ['systemd-importd', 'systemd-import and backends']
+  journal: ['systemd-journald', 'journalctl']
+  journal-remote: ['systemd-journal-remote', 'systemd-journal-upload', 'systemd-journal-gatewayd']
+  kernel-install: ['kernel-install (incl. any plugins)']
+  logind: ['systemd-logind', 'loginctl', 'pam_systemd']
+  machined: ['systemd-machined', 'machinectl']
+  modules-load: ['systemd-modules-load']
+  network: ['systemd-networkd', 'networkctl', 'systemd-networkd-wait-online', '.network files']
+  nspawn: ['systemd-nspawn']
+  oomd: ['systemd-oomd', 'oomctl']
+  pid1: ['systemd']
+  portabled: ['systemd-portabled', 'portablectl']
+  pstore: ['systemd-pstore']
+  repart: ['systemd-repart']
+  resolve: ['systemd-resolved', 'resolvectl', 'nss-resolve']
+  rfkill: ['systemd-rfkill']
+  rpm: ['rpm scriptlets']
+  run: ['systemd-run']
+  sd-boot: ['the bootloader itself', 'bootctl']
```

Comment:
There is no `sd-boot`. We use `boot` for anything in `bootctl`, `systemd-boot` and `systemd-stub` (and those should be used as components above).

Reply:
Maybe let's just rename `boot` to `sd-boot`? `boot` is a big too generic, users might think that this about anything related to boot.

Reply:
OK, nobody is protesting, so I'm doing the rename.